Build is broken

Facing build issue with the latest version
$ sudo make
mkdir -p build/libcxxabi
cd build/libcxxabi; cmake -G "Unix Makefiles" \
	-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
	-DCMAKE_CXX_COMPILER_WORKS=ON \
	-DCMAKE_C_COMPILER_WORKS=ON \
	-DLIBCXXABI_ENABLE_EXCEPTIONS:BOOL=OFF \
	-DLIBCXXABI_ENABLE_SHARED:BOOL=OFF \
	-DLIBCXXABI_HAS_PTHREAD_API:BOOL=ON \
	-DCXX_SUPPORTS_CXX11=ON \
	-DLLVM_COMPILER_CHECKED=ON \
	-DCMAKE_BUILD_TYPE=RelWithDebugInfo \
	-DLIBCXXABI_LIBCXX_PATH=/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxx \
	-DLIBCXXABI_LIBCXX_INCLUDES=/opt/wasi-sdk/share/sysroot/include/c++/v1 \
	-DLLVM_CONFIG_PATH=/media/ExtendedDrive/wasi-sdk/build/llvm/bin/llvm-config \
	-DCMAKE_TOOLCHAIN_FILE=/media/ExtendedDrive/wasi-sdk/wasi-sdk.cmake \
	-DWASI_SDK_PREFIX=/opt/wasi-sdk \
	-DCMAKE_C_FLAGS="-fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m" \
	-DCMAKE_CXX_FLAGS="-fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m" \
	-DUNIX:BOOL=ON \
	--debug-trycompile \
	/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi
debug trycompile on
System is unknown to cmake, create:
Platform/Wasm to use this system, please send your config file to cmake@www.cmake.org so it can be added to cmake
Your CMakeCache.txt file was copied to CopyOfCMakeCache.txt. Please send that file to cmake@www.cmake.org.
-- Configuring for standalone build.
-- Found LLVM_CONFIG_PATH as /media/ExtendedDrive/wasi-sdk/build/llvm/bin/llvm-config
-- Linker detection: LLD
-- Building with -fPIC
CMake Warning at CMakeLists.txt:476 (message):
  The libc++abi tests aren't valid when libc++abi is built standalone (i.e.
  outside of llvm/projects/libcxxabi ) and is built without a shared library.
  Either build a shared library, build libc++abi at the same time as you
  build libc++, or do without testing.  No check target will be available!


-- Configuring done
-- Generating done
-- Build files have been written to: /media/ExtendedDrive/wasi-sdk/build/libcxxabi
cd build/libcxxabi; make -j 8 install
make[1]: Entering directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
/usr/local/bin/cmake -S/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi -B/media/ExtendedDrive/wasi-sdk/build/libcxxabi --check-build-system CMakeFiles/Makefile.cmake 0
/usr/local/bin/cmake -E cmake_progress_start /media/ExtendedDrive/wasi-sdk/build/libcxxabi/CMakeFiles /media/ExtendedDrive/wasi-sdk/build/libcxxabi/CMakeFiles/progress.marks
make -f CMakeFiles/Makefile2 all
make[2]: Entering directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
make -f src/CMakeFiles/cxxabi_objects.dir/build.make src/CMakeFiles/cxxabi_objects.dir/depend
make[3]: Entering directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi && /usr/local/bin/cmake -E cmake_depends "Unix Makefiles" /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src /media/ExtendedDrive/wasi-sdk/build/libcxxabi /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src/CMakeFiles/cxxabi_objects.dir/DependInfo.cmake --color=
make[3]: Leaving directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
make -f src/CMakeFiles/cxxabi_objects.dir/build.make src/CMakeFiles/cxxabi_objects.dir/build
make[3]: Entering directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
[  5%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_aux_runtime.cpp.o
[ 10%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_default_handlers.cpp.o
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_aux_runtime.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_aux_runtime.cpp
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_default_handlers.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_default_handlers.cpp
[ 15%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_exception_storage.cpp.o
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_exception_storage.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_exception_storage.cpp
[ 21%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_demangle.cpp.o
[ 26%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_guard.cpp.o
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_demangle.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_demangle.cpp
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_guard.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_guard.cpp
[ 31%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_handlers.cpp.o
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_handlers.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_handlers.cpp
[ 36%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_vector.cpp.o
[ 42%] Building CXX object src/CMakeFiles/cxxabi_objects.dir/cxa_unexpected.cpp.o
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_vector.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_vector.cpp
cd /media/ExtendedDrive/wasi-sdk/build/libcxxabi/src && /opt/wasi-sdk/bin/clang++ --target=wasm32-unknown-wasi --sysroot=/opt/wasi-sdk/share/sysroot  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LIBCPP_DISABLE_EXTERN_TEMPLATE -D_LIBCPP_ENABLE_CXX17_REMOVED_UNEXPECTED_FUNCTIONS -D_LIBCPP_HAS_THREAD_API_PTHREAD -D_LIBCXXABI_BUILDING_LIBRARY -D_LIBCXXABI_NO_EXCEPTIONS -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/include  -fdebug-prefix-map=/media/ExtendedDrive/wasi-sdk=wasisdk://v3.6g8eff94c85cdc+m -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++11 -w -ffunction-sections -fdata-sections  -fPIC   -nostdinc++ -Werror=return-type -W -Wall -Wchar-subscripts -Wconversion -Wmismatched-tags -Wmissing-braces -Wnewline-eof -Wunused-function -Wshadow -Wshorten-64-to-32 -Wsign-compare -Wsign-conversion -Wstrict-aliasing=2 -Wstrict-overflow=4 -Wunused-parameter -Wunused-variable -Wwrite-strings -Wundef -Wno-error -WX- -pedantic -fstrict-aliasing -fno-exceptions -D_DEBUG -D_LIBCPP_BUILD_STATIC -std=c++11 -o CMakeFiles/cxxabi_objects.dir/cxa_unexpected.cpp.o -c /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_unexpected.cpp
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_default_handlers.cpp:12:10: fatal error: 'stdexcept' file not found
#include <stdexcept>
         ^~~~~~~~~~~
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_handlers.cpp:13:10: fatal error: 'stdexcept' file not found
#include <stdexcept>
         ^~~~~~~~~~~
In file included from /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_demangle.cpp:18:
In file included from /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/demangle/ItaniumDemangle.h:23:
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/demangle/StringView.h:15:10: fatal error: 'algorithm' file not found
#include <algorithm>
         ^~~~~~~~~~~
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_guard.cpp:13:10: fatal error: '__threading_support' file not found
#include <__threading_support>
         ^~~~~~~~~~~~~~~~~~~~~
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_aux_runtime.cpp:14:10: fatal error: 'new' file not found
#include <new>
         ^~~~~
1 error generated.
1 error generated.
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_unexpected.cpp:10:10: fatal error: 'exception' file not found
#include <exception>
         ^~~~~~~~~~~
src/CMakeFiles/cxxabi_objects.dir/build.make:117: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_guard.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_guard.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
src/CMakeFiles/cxxabi_objects.dir/build.make:65: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_aux_runtime.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_aux_runtime.cpp.o] Error 1
In file included from /media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_exception_storage.cpp:14:
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_exception.hpp:17:10: fatal error: 'exception' file not found
#include <exception> // for std::unexpected_handler and std::terminate_handler
         ^~~~~~~~~~~
1 error generated.
1 error generated.
src/CMakeFiles/cxxabi_objects.dir/build.make:130: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_handlers.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_handlers.cpp.o] Error 1
1 error generated.
src/CMakeFiles/cxxabi_objects.dir/build.make:78: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_default_handlers.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_default_handlers.cpp.o] Error 1
src/CMakeFiles/cxxabi_objects.dir/build.make:143: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_unexpected.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_unexpected.cpp.o] Error 1
1 error generated.
src/CMakeFiles/cxxabi_objects.dir/build.make:104: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_exception_storage.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_exception_storage.cpp.o] Error 1
/media/ExtendedDrive/wasi-sdk/src/llvm-project/libcxxabi/src/cxa_vector.cpp:16:10: fatal error: 'exception' file not found
#include <exception>        // for std::terminate
         ^~~~~~~~~~~
1 error generated.
src/CMakeFiles/cxxabi_objects.dir/build.make:156: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_vector.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_vector.cpp.o] Error 1
1 error generated.
src/CMakeFiles/cxxabi_objects.dir/build.make:91: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/cxa_demangle.cpp.o' failed
make[3]: *** [src/CMakeFiles/cxxabi_objects.dir/cxa_demangle.cpp.o] Error 1
make[3]: Leaving directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
CMakeFiles/Makefile2:195: recipe for target 'src/CMakeFiles/cxxabi_objects.dir/all' failed
make[2]: *** [src/CMakeFiles/cxxabi_objects.dir/all] Error 2
make[2]: Leaving directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
Makefile:120: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/media/ExtendedDrive/wasi-sdk/build/libcxxabi'
Makefile:97: recipe for target 'build/libcxxabi.BUILT' failed
make: *** [build/libcxxabi.BUILT] Error 2

--- a/Makefile
+++ b/Makefile
@@ -118,8 +118,8 @@ build/libcxxabi.BUILT: build/libcxx.BUILT build/llvm.BUILT
 		-DLLVM_CONFIG_PATH=$(ROOT_DIR)/build/llvm/bin/llvm-config \
 		-DCMAKE_TOOLCHAIN_FILE=$(ROOT_DIR)/wasi-sdk.cmake \
 		-DWASI_SDK_PREFIX=$(PREFIX) \
-		-DCMAKE_C_FLAGS="$(DEBUG_PREFIX_MAP)" \
-		-DCMAKE_CXX_FLAGS="$(DEBUG_PREFIX_MAP)" \
+		-DCMAKE_C_FLAGS="$(DEBUG_PREFIX_MAP) -I$(PREFIX)/share/sysroot/include" \
+		-DCMAKE_CXX_FLAGS="$(DEBUG_PREFIX_MAP) -I$(PREFIX)/share/sysroot/include/c++/v1" \
 		-DUNIX:BOOL=ON \
 		--debug-trycompile \
 		$(LLVM_PROJ_DIR)/libcxxabi

